wrapper/flags.go

```diff
@@ -66,7 +67,7 @@ func ParseArgs(args []string, custom map[string]bool) (customFlags FlagValues, p
 			protocFlags = append(protocFlags, arg)
 			nextIsFlag = false
 			if nextIsImportDir {
-				importDirs = append(importDirs, strings.Split(arg, ":")...)
+				importDirs = append(importDirs, strings.Split(arg, fmt.Sprintf("%c", os.PathListSeparator))...)
```

Comment:
`string(os.PathListSeparator)` is fine ([ref](https://golang.org/ref/spec#Conversions))